internal/controllers/rest_controller.go

```diff
@@ -76,19 +134,67 @@ func (rc RESTController) OIDCProviderCallback(w http.ResponseWriter, r *http.Req
 	}
 
 	// If the flow is secure, a JWT will be created...
-	jwt, err := rc.oidcService.CreateJWT(&token.CustomClaims)
+	jwt, err := rc.oidcService.Create(&token.CustomClaims)
 	if err != nil {
 		rc.logger.Error("an error occurred while creating a JWT. Error: " + err.Error())
 		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
Why you aren't using `models.ErrorResponse` here?

Reply:
ops! thanks Renato.